src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -26,27 +26,35 @@ function clone<T extends GPUBindGroupLayoutDescriptor>(descriptor: T): T {
 
 export const g = makeTestGroup(ValidationTest);
 
-g.test('some_binding_index_was_specified_more_than_once').fn(async t => {
-  const goodDescriptor = {
-    entries: [
-      { binding: 0, visibility: GPUShaderStage.COMPUTE, type: 'storage-buffer' as const },
-      { binding: 1, visibility: GPUShaderStage.COMPUTE, type: 'storage-buffer' as const },
-    ],
-  };
+g.test('some_binding_index_was_specified_more_than_once')
```

Comment:
I forgot about this on your last review PR, but for these files I'd like to also trim down the test names as we add descriptions. These test names were written before we had descriptions, so they were descriptive out of necessity. But now they can be shorter (which makes the names easier to copy, logs less verbose... and helps avoid path length issues on windows).

This one for example could be `'duplicate_binding'`